app/assets/admin/js/settings.js

```diff
@@ -961,6 +962,15 @@ jQuery( document ).ready( function ( $ ) {
 			downloadAnchorNode.remove();
 		} );
 	} );
+
+	jQuery( '#rtm-export-data-button' ).click( function(){
```

Comment:
Isn' this anchor tag?

Reply:
Yes. But we wanted to redirection to `Export Personal Data` and `Erase Personal Data` page on click. SO we have made it Button link.
@chandrapatel